src/test/fuzz/fuzz.cpp

```diff
@@ -50,7 +46,8 @@ extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv)
 // the main(...) function.
 __attribute__((weak))
 #endif
```

Comment:
What is this used for? I don't think it is possible to run the fuzzers on windows

Reply:
Removed CygWin workaround.